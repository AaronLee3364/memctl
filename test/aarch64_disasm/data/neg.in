fffffff0070c0000 4b1803e8 4b481fe8
