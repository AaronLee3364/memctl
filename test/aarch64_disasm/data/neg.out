fffffff0070c0000  NEG     W8, W24
fffffff0070c0004  NEG     W8, W8, LSR #7
